styx-common/src/main/java/com/spotify/styx/storage/BigtableStorage.java

```diff
@@ -119,6 +121,11 @@ void writeEvent(SequenceEvent sequenceEvent) throws IOException {
         scan.setStartRow(Bytes.toBytes(offsetInstance.toKey() + '#'));
       }
 
+      // If `tail` is:
+      //   true: Return the _latest_ instances by scanning over _all_ the rows for this workflow (from offset).
+      //   false: Return the _newest_ instances by only scanning over as many rows as necessary.
```

Comment:
What is the difference between latest/newest here? To me it sounds like they both refer to the most recent instances based on parameter, but I suspect that isn't the case? Just wondering if there are more descriptive words we could use.

Reply:
That was a typo, good catch 👍